ioctl/newcmd/account/accountdelete_test.go

```diff
@@ -24,14 +28,21 @@ func TestNewAccountDelete(t *testing.T) {
 	client.EXPECT().SelectTranslation(gomock.Any()).Return("mockTranslationString",
 		config.English).AnyTimes()
 
-	client.EXPECT().GetAddress(gomock.Any()).Return("io1kuz56knh9g7x3ht36t2pgfd8fz5g6rxrh5a8dn", nil)
-
-	ks := keystore.NewKeyStore(config.ReadConfig.Wallet,
+	ks := keystore.NewKeyStore("./",
```

Comment:
let's create the keystore under a temp file folder, like in other tests

Reply:
and delete the temp file, like in other tests

```
defer func() {
		testutil.CleanupPath(t, testPath)
	}()
```

Reply:
How can the file name of tmp keystore be acquired in this test?

Reply:
```
testPath, err := testutil.PathOfTempFile("test-delete")
r.NoError(err)
```
testPath is the filename